doc/ref_arch/kubernetes/chapters/chapter02.md

```diff
@@ -23,7 +28,120 @@
+
+<a name="2.2.4"></a>
+### 2.2.4 Instance Capabilities Mapping
+
+| Attribute | Description | Value | Supported |
+|-----------|---------------------------|-------|-------|
+| e.nfvi.res.cap.001 | Max number of vCPU that can be assigned to a single VNFC | 16 | Y |
+| e.nfvi.res.cap.002 | Max memory in MB that can be assigned to a single VNFC by NFVI | 32 GB | Y |
+| e.nfvi.res.cap.003 | Max storage in GB that can be assigned to a single VNFC by NFVI | 320 GB | Y |
+| e.nfvi.res.cap.004 | # Connection Points | 6 | Y |
+| e.nfvi.res.cap.005 | Total instance (persistent) storage (GB | 300 GB | Y |
+| e.nfvi.per.cap.001 | CPU pinning support | | Y |
+| e.nfvi.per.cap.002 | NUMA support | | Y |
+| e.nfvi.per.cap.003 | IPSec Acceleration | | Yes (if offered |
+| e.nfvi.per.cap.004 | Crypto Acceleration | | Yes (if offered |
+| e.nfvi.per.cap.005 | Transcoding Acceleration | | Yes (if offered |
+| e.nfvi.per.cap.006 | Programmable Acceleration | | Yes (if offered |
+| e.nfvi.per.cap.007 | Enhanced Cache Management | | X (if offered |
+| e.nfvi.mon.cap.001 | Monitoring of L2-7 data | | N<sup>1)</sup> |
+| i.nfvi.sla.cap.001 | CPU overbooking | 1:1, 1:4 | Y |
+| i.nfvi.sla.cap.002 | vNIC QoS | | Y |
+| i.nfvi.per.cap.001 | Huge page support |  | Y |
+| i.nfvi.mon.cap.001 | Monitor host CPU usage |  | Y |
+| i.nfvi.mon.cap.002 | Monitor virtual compute CPU usage | | Y |
+| i.nfvi.mon.cap.003 | Monitor host CPU utilization | | Y |
+| i.nfvi.mon.cap.004 | Monitor virtual compute CPU utilization | | Y |
+| i.nfvi.mon.cap.007 | Monitor external storage capacity | | Y |
+| nfvi.com.cfg.001 | CPU allocation ratio  | 4:1, 1:1 | Y |
+| nfvi.com.cfg.002 | NUMA awareness | | Y |
+| nfvi.com.cfg.003 | CPU pinning capability | | Y |
+| nfvi.com.cfg.004 | Huge Pages  | | Y |
+| nfvi.stg.cfg.001 | Catalogue storage Types | | Y |
+| nfvi.stg.cfg.002 | Storage Block |  | Y |
+| nfvi.stg.cfg.003 | Storage Object | | N<sup>2)</sub> |
+| nfvi.stg.cfg.004 | Storage with replication | | Y |
+| nfvi.stg.cfg.005 | Storage with encryption | | Y |
+| nfvi.stg.acc.cfg.001 | Storage IOPS oriented | | Y |
+| nfvi.stg.acc.cfg.002 | Storage capacity oriented | | Y |
+| nfvi.net.cfg.001 | vNIC interface | | N<sup>3)</sup>|
+| nfvi.net.cfg.002 | Overlay protocol | | Y<sup>4)</sup>|
+| nfvi.net.cfg.003 | NAT | | |
+| nfvi.net.cfg.004 | Security Group | | |
+| nfvi.net.cfg.005 | SFC support | | |
+| nfvi.net.cfg.006 | Traffic patterns symmetry | | |
+| nfvi.net.acc.cfg.001 | vSwitch optimisation | | N <sup>5)</sup>|
+| nfvi.net.acc.cfg.002 | Support of HW offload | |Y, support of SmartNic |
+| nfvi.net.acc.cfg.003 | Crypto acceleration | | Y |
+| nfvi.net.acc.cfg.004 | Crypto Acceleration Interface | | ? |
+| nfvi.hw.cpu.cfg.001 | Number of CPU (Sockets) |  2 | Y |
+| nfvi.hw.cpu.cfg.002 | Number of Cores per CPU | 20 | Y |
+| nfvi.hw.cpu.cfg.003 | NUMA | | Y |
+| nfvi.hw.cpu.cfg.004 | Simultaneous Multithreading/Hyperthreading (SMT/HT) | | Y |
+| nfvi.hw.cac.cfg.001 | GPU | | Y |
```

Comment:
should this be "optional"

Reply:
It is mandatory according to RM section [5.4.1.1 Compute Acceleration Hardware Specifications](https://github.com/cntt-n/CNTT/blob/0e4a3d6720ab021b83e3b5f94711c469c479b477/doc/ref_model/chapters/chapter05.md#5411-compute-acceleration-hardware-specifications)

Reply:
@CsatariGergely Issue #875 opened against RM Ch 05.

Reply:
Thanks. Let's sync to here once #875 is closed.